doc/classes/ImageTexture.xml

```diff
@@ -4,10 +4,38 @@
 		A [Texture2D] based on an [Image].
 	</brief_description>
 	<description>
-		A [Texture2D] based on an [Image]. Can be created from an [Image] with [method create_from_image].
-		[b]Note:[/b] The maximum image size is 16384×16384 pixels due to graphics hardware limitations. Larger images will fail to import.
+		A [Texture2D] based on an [Image]. For an image to be displayed, an [ImageTexture] has to be created from it using the [method create_from_image] method:
+		[codeblock]
+		var texture = ImageTexture.new()
+		var image = Image.new()
+		image.load("res://icon.png")
+		texture.create_from_image(image)
+		$Sprite2D.texture = texture
+		[/codeblock]
+		This way, textures can be created at run-time by loading images both from within the editor and externally.
+		[b]Warning:[/b] Prefer to load imported textures over loading them from within the filesystem dynamically, as it may not work in exported projects:
+		[codeblock]
+		var texture = load("res://icon.png")
+		$Sprite2D.texture = texture
+		[/codeblock]
+		This is because [method Image.load] relies on OS-specific file access methods that bypass the virtual filesystem implemented in the engine to load imported resources, so [method Image.load] should only be used in the editor or in cases when you need to load external images at run-time, such as images located at the [code]user://[/code] directory.
+		If you'd still like to load an image file just like any other [Resource], import it as an [Image] resource instead, and then load it normally using the [method @GDScript.load] method:
+		[codeblock]
+		var image = load("res://icon.png")
+		assert(image is Image)
+		assert(not (image is ImageTexture))
+		[/codeblock]
+		But do note that the image data can still be retrieved from an imported texture as well using the [method Texture2D.get_data] method, which returns a copy of the data:
+		[codeblock]
+		var texture = load("res://icon.png")
+		var image = texture.get_data()
+		assert(image is Image)
+		[/codeblock]
+		An [ImageTexture] is not meant to be operated from within the editor interface directly, and is mostly useful for rendering images on screen dynamically via code. If you need to generate images procedurally from within the editor, consider saving and importing images as custom texture resources implementing a new [EditorImportPlugin].
+		[b]Note:[/b] The maximum texture size is 16384×16384 pixels due to graphics hardware limitations.
 	</description>
 	<tutorials>
+		<link title="Importing images">https://docs.godotengine.org/en/latest/getting_started/workflow/assets/importing_images.html</link>
```

Comment:
It moved: https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/importing_images.html